internal/resume/templates/latex.go

```diff
@@ -111,6 +111,7 @@ var latexDocument = `
 
 %==== Skills ====%
 \header{Skills}
+\vspace{1mm}
```

Comment:
Might as well add this to the project section as well.

Reply:
sorry i forgot to add it to my tests. I need to learn to use automation before pushing.